src/cli/router_cli_hotspot_reputation.erl

```diff
@@ -80,6 +96,69 @@ hotspot_reputation_reset(["hotspot_reputation", "reset", B58], [], []) ->
 hotspot_reputation_reset([_, _, _], [], []) ->
     usage.
 
+hotspot_reputation_sc(["hotspot_reputation", "sc"], [], Flags) ->
+    TimeOverAvg = proplists:get_value(over, Flags, 10),
+    ReputationOver = proplists:get_value(rep, Flags, 0),
+    ActiveSCs = maps:values(blockchain_state_channels_server:get_actives()),
+    Avg = maps:from_list(
+        lists:map(
+            fun({SC, _SCState, _Pid}) ->
+                TotalDcs = blockchain_state_channel_v1:total_dcs(SC),
+                case erlang:length(blockchain_state_channel_v1:summaries(SC)) of
+                    0 ->
+                        {blockchain_state_channel_v1:name(SC), 0};
+                    Actors ->
+                        {blockchain_state_channel_v1:name(SC), TotalDcs / Actors}
+                end
+            end,
+            ActiveSCs
+        )
+    ),
+    HotspotList = lists:concat(
+        lists:map(
```

Comment:
No need to change anything.  Just calling this out as potential threshold for when to introduce intermediate variables for params rather than inlined values; whereas, the previous use of `lists:maps()` a few lines above is short enough to keep as-is.

Mind you, I have lots of old code (especially in Common Lisp) where I didn't introduce enough intermediate vars, and after revisiting that code years or months later is why I'm mentioning this now.

Thoughts?  we can discuss in #router channel.